pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -222,7 +235,9 @@ func (f *framework) QueueSortFunc() LessFunc {
 // anything but Success. If a non-success status is returned, then the scheduling
 // cycle is aborted.
 func (f *framework) RunPreFilterPlugins(
-	state *CycleState, pod *v1.Pod) *Status {
+	state *CycleState, pod *v1.Pod) (status *Status) {
+	startTime := time.Now()
+	defer recordExtensionPointDuration(startTime, preFilter, &status)
```

Comment:
Have you tested it? I think `recordExtensionPointDuration` will always get `status` as nil, isn't it?

Reply:
I don't think using named return variable is a good practice here.

Reply:
yes please take a look at the unit tests.
The purpose of having a named return variable is to make the `defer` statement easy to execute. Otherwise we have to make sure the `recordExtensionPointDurationc` called properly in various control branches, which is more error prone.